components/service-waiter/cmd/root.go

```diff
@@ -100,22 +100,6 @@ func getTimeout() time.Duration {
 	return timeout
 }
 
-// fail ends the waiting process propagating the message on its way out
```

Comment:
why did we do like that before and not just used Fatal? Is it to have nice output on crash loop events in UI and kubectl describe? Should we keep it?

Reply:
This function will make build failed since no where use it